wandb/sdk/wandb_settings.py

```diff
@@ -493,6 +493,9 @@ class Settings(BaseModel, validate_assignment=True):
     silent: bool = False
     """Flag to suppress all output."""
 
+    disable_histogram: bool = False
```

Comment:
What we show at the end isn't a histogram; `disable_sparkline` would be a better name